scout/commands/export/variant.py

```diff
@@ -51,7 +51,12 @@ def verified(context, collaborator, test, outpath=None):
         LOG.warning('There are no verified variants for institute {} in database!'.format(collaborator))
         return None
 
-    document_lines = export_verified_variants(verified_vars)
+    unique_callers = set()
+    for var_type, var_callers in CALLERS.items():
+        for caller in var_callers:
+            unique_callers.add(caller.get('id'))
+
+    document_lines = export_verified_variants(verified_vars, unique_callers)
```

Comment:
A little bit OT I suppose? Well, you are the expert here @chiara: looks good to me, if you confirm this is intentional.